src/containers/Tokens/index.js

```diff
@@ -40,8 +53,13 @@ const Tokens = props => {
       });
   }, []);
 
-  return error ? (
-    Tokens.renderError(error)
+  function renderError(errorInfo) {
+    const { result, message } = errorInfo;
+    return <NoMatch title={result} hints={[message]} />;
+  }
+
+  return error || isError ? (
+    renderError(errorInfoRef.current)
```

Comment:
Why are we using a `ref` for this? We can just only render it if there's an error